hack/lib/init.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# This script is meant to be the entrypoint for OpenShift Bash scripts to import all of the support
+# libraries at once in order to make Bash script preambles as minimal as possible. This script recur-
+# sively `source`s *.sh files in this directory tree. As such, no files should be `source`ed outside
+# of this script to ensure that we do not attempt to overwrite read-only variables.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+API_GROUP_VERSIONS="apps/v1 authorization/v1 build/v1 image/v1 network/v1 oauth/v1 project/v1 quota/v1 route/v1 security/v1 template/v1 user/v1"
```

Comment:
nit: list with newlines would be easier to read.

Reply:
👍 good point, updated both envs.